packages/adblocker/src/filters/cosmetic.ts

```diff
@@ -795,8 +795,16 @@ export default class CosmeticFilter implements IFilter {
       .slice(1)
       .map((part) => {
         if (
-          (part.startsWith(`'`) && part.endsWith(`'`)) ||
-          (part.startsWith(`"`) && part.endsWith(`"`))
+          (part.startsWith(`'`) &&
+            part.endsWith(`'`) &&
+            part.length >= 2 &&
+            !part.slice(1, -1).match(/[^\\]'/) &&
+            !part.slice(1, -1).includes(',')) ||
```

Comment:
This condition is invalid because it can have comma inside validly. See https://github.com/ghostery/adblocker/pull/5105/files#r2283885167 and https://github.com/ghostery/adblocker/pull/5105/files#r2283883454